templates/vm.conf.j2

```diff
@@ -42,10 +42,10 @@ ramdisk                 = "{{ vm.initrd|default(xen_vman_default_initrd) }}"
 kernel                  = "{{ vm.kernel|default(xen_vman_default_kernel) }}"
 {% if vm.storage_type|default(xen_vman_default_storage_type) == "nfs" and vm.cmdline is not defined %}
 cmdline                 = "root=/dev/nfs nfsroot=
-                           {{- xen_vman_nfsroot_base }}{{ vm.org }}/{{ vm.host }}-root ip=
-                           {{- vm.connection_ip|default(vm['interfaces'][0]['ip']|ipaddr('address/prefix'))|default('dhcp') }} {{ vm.cmd|default(xen_vman_default_cmdline) }}"
+                           {{- xen_vman_nfsroot_base }}{{ vm.org }}/{{ vm.name }}-root ip=
+                           {{- vm.connection_ip|default(vm['interfaces'][0]['ip']|ipaddr('address/prefix'))|default('dhcp') }}"
 {% elif vm.storage_type|default(xen_vman_default_storage_type) == "iscsi" and vm.cmdline is not defined %}
-cmdline                 = "root=/dev/xvda rootfstype={{ vm.filesystem|default(xen_vman_default_filesystem) }} rw {{ vm.cmd|default(xen_vman_default_cmdline) }}"
+cmdline                 = "root=/dev/xvda rootfstype={{ vm.filesystem|default(xen_vman_default_filesystem) }} rw"
```

Comment:
I meant that it should be `vm.cmdline` instead of `vm.cmd`. Now there is only an cmdline appended if the type is neither iscsi nor nfs. If set, it will use the `else` branch and won't set the iscsi or nfs specific options. I thought `vm.cmdline` was intended as an amendment though.